src/models/Orders.js

```diff
@@ -0,0 +1,105 @@
+const mongoose = require('mongoose');
+
+// mongoose schema for the individual order items
+const orderItemSchema = new mongoose.Schema({
+    // The album field references the Album model and is required
+  album: {
+    type: mongoose.Schema.Types.ObjectId,
+    ref: 'Album', // Referencing the Album model
+    required: true,
+  },
+  // The quantity field specifies the number of items in the order
+  quantity: {
+    type: Number,
+    required: true, // Quantity is required
+    min: 1, // Minimum quantity allowed is 1
+  },
+});
+
+// Create the OrderItem model using the defined schema
+const OrderItem = mongoose.model('OrderItem', orderItemSchema);
```

Comment:
It's fine to leave this; but we probably won't ever need to create an OrderItem directly/individually apart from an `Order` so we don't _need_ to declare it as a model. But it doesn't hurt!